parastell/invessel_build.py

```diff
@@ -199,6 +202,10 @@ def populate_surfaces(self):
             expand_list(self.radial_build.poloidal_angles, self.num_rib_pts)
         )
 
+        # Store actual number of ribs and rib points
+        self._num_ribs = len(self._toroidal_angles_exp)
+        self._num_rib_pts = len(self._poloidal_angles_exp)
```

Comment:
How do these relate to `self.num_ribs` and `self.num_rib_pts`?  I' concerned that we have both defined and they could be different?

Reply:
Perhaps it would be more appropriate to change these variable names, but I defined these separately because it can be the case that `self.num_ribs` and `self._num_ribs`, etc. don't have the same value, due to the way we expand the angle lists. For the MOAB routine, it's important we know the actual number of ribs and rib points, rather than what the user defined.